src/main/scala/uk/co/gresearch/spark/diff/DiffOptions.scala

```diff
@@ -27,19 +32,41 @@ package uk.co.gresearch.spark.diff
  * @param changeDiffValue value in diff column for changed rows
  * @param deleteDiffValue value in diff column for deleted rows
  * @param nochangeDiffValue value in diff column for un-changed rows
+ * @param nullOutValidData if true, all cells which are equal (or in a given tolerance range) are replaced with null to increase visibility of erroneous data
+ * @param ignoreColumns an optional list of columns which are dropped from both sides before comparing
+ * @param datatypeComparators an optional map of datatypes with the applicable comparator, overwriting the default '<=>' comparator
+ * @param columnComparators an optional map of column names with the applicable comparator, overwriting the datatype comparators
+ * @param joinType  Type of the join to perform. Default `full_outer`, should usually not be changed.
```

Comment:
Yet another feature, separate PR! Not sure about useful use-cases for this though.